tests/engines/test_gradient_accumulation.py

```
@@ -0,0 +1,343 @@
+    # ---- integration (require ignite) ----
+
+    @unittest.skipUnless(has_ignite, "Requires pytorch-ignite")
+    def test_integration_gradient_equivalence(self) -> None:
+        """Accumulated gradients over N mini-batches equal one large-batch step."""
+        from monai.engines import SupervisedTrainer
+
+        torch.manual_seed(42)
+        acc_steps, lr = 4, 0.1
+        batches = [{CommonKeys.IMAGE: torch.randn(1, 4), CommonKeys.LABEL: torch.randn(1, 1)} for _ in range(acc_steps)]
+
+        ref_model, test_model, ref_opt, test_opt, init_weight = _make_model_pair(lr)
```

Ruff (0.15.2)

[warning] Unpacked variable `init_weight` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)